code/game/turfs/simulated/minerals.dm

```diff
@@ -400,6 +400,35 @@
 	baseturf = /turf/simulated/floor/plating/lava/smooth/lava_land_surface
 	defer_change = 1
 
+//Asteroid turfs
+/turf/simulated/mineral/asteroid
```

Comment:
Can I ask what is wrong with using the existing mineral turfs?

Reply:
Primarily aesthetic purposes. 
The gray ash walls of lavaland being used for all space asteroids has bugged me for a long time as it just seems off, especially combined with the sandy ground turfs used on those space structures.